cmd/neofs-node/fs_windows.go

```diff
@@ -0,0 +1,37 @@
+//go:build windows
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"golang.org/x/sys/windows"
+)
+
+func totalBytes(paths []string) (uint64, error) {
+	var total uint64 // bytes
+	diskMap := make(map[string]struct{}, len(paths))
+
+	for _, path := range paths {
+		var availB uint64
+		var totalB uint64
+		var freeTotalB uint64
+		var pathP = windows.StringToUTF16Ptr(path)
+
+		err := windows.GetDiskFreeSpaceEx(pathP, &availB, &totalB, &freeTotalB)
+		if err != nil {
+			return 0, fmt.Errorf("get disk stat by '%s' path: %w", path, err)
+		}
+
+		disk := filepath.VolumeName(path)
```

Comment:
may be checked before syscall, or u wanna ensure that path is "ok"?

Reply:
it makes sense for unix (and is the only way) but senseless for windows, moved above the syscall